acceptance/lib/helper.rb

```diff
@@ -303,9 +303,9 @@ def hup_server(host = master, timeout = 30)
       begin
         response = https_request(url, 'GET', cert, key)
       rescue StandardError => e
-        expected_errors = [ Errno::ECONNREFUSED, Errno::ECONNRESET, 
+        expected_errors = [ Errno::ECONNREFUSED, Errno::ECONNRESET,
                             OpenSSL::SSL::SSLError ]
-        if !expected_errors.include?e.class 
+        if !expected_errors.include?e.class
```

Comment:
If it looked like this (and this still works), it would seem more readable to me:

~~~ruby
if !expected_errors.include?(e.class)
~~~

But not a big deal to me either way.